src/core/AutoRest.Extensions/ParameterGroupExtensionHelper.cs

```diff
@@ -110,9 +110,7 @@ private static string GenerateParameterGroupModelText(IEnumerable<Method> method
             List<Method> methodList = methodsWhichUseGroup.ToList();
             if (methodList.Count == 1)
             {
-                Method method = methodList.Single();
-                return string.Format(CultureInfo.InvariantCulture, "Additional parameters for the {0} operation.",
-                    createOperationDisplayString(method.MethodGroup.Name.ToPascalCase(), method.Name));
+                return "Additional parameters for "+ methodList.Single().Name + " operation.";
```

Comment:
If by above you mean `AzureExtensions.cs`, for some reason looks like the `method` variable there has `name` in the form of operationId itself while in `ParameterGroupExtensionHelper.cs`, the `name` property denotes the `{1}` in `{0}_{1}` in an operationId.
We could probably investigate where these diverge but looks like overkill for such a small change.